templates/embedding/base_embedding.html

```diff
@@ -0,0 +1,14 @@
+{% extends 'base_index.html' %}
+
+{% block meta_copydoc %}
+  https://drive.google.com/drive/folders/19PBDauuZAlJPizPte3Es-Rc3FFmp8Xt2
+{% endblock meta_copydoc %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
+{% block outer_content %}
+
+  {% block content %}{% endblock %}
+{% endblock %}
```

Comment:
The base file might not be needed since there is no outer_content in `base_index.html`,
So you might be good with just adding 
```
{% block body_class %}
  is-paper
{% endblock body_class %}
``` to rest of pages